src/mixins/ClassNameMixin.jsx

```diff
@@ -42,7 +43,7 @@ var ClassNameMixin = {
     var mergedClassName = mergeClassSetObjects(componentClassName, className);
     mergedClassName = mergeClassSetObjects(mergedClassName, propClassName);
 
-    return React.addons.classSet(mergedClassName);
+    return classNames(mergedClassName);
```

Comment:
Now that we're using classNames, we no longer need react/addons and we can change:

```js
var React = require('react/addons');
```
to
```js
var React = require('react');
```